initializr-generator/src/main/java/io/spring/initializr/generator/ProjectGenerator.java

```diff
@@ -74,6 +75,9 @@
 
 	private static final Version VERSION_2_0_0_M6 = Version.parse("2.0.0.M6");
 
+	private static final String KUBERNETES_FILE = "kubernetes.yaml";
```

Comment:
I notice most of `yaml` file keep `.yml` as suffix, do we need to align this ?

Reply:
I also noticed this, it seems that K8s use yaml as kind of convention. I'll check this later